src/Symfony/Bridge/Twig/Resources/views/Form/bulma_0_layout.html.twig

```diff
@@ -0,0 +1,97 @@
+{% extends "form_div_layout.html.twig" %}
+
+{%- block form_widget_compound -%}
+    <div {{ block('widget_container_attributes') }} class="columns is-multiline">
```

Comment:
The classes should be injecting into the container attributes rather than being added separately. Otherwise, we get 2 separate `class` attributes when injecting a class in the `attr` (which is a common use case). And duplicate attributes won't work (one of them will win)